low/low.c

```diff
@@ -108,14 +132,19 @@ void recv(uint8_t port, uint16_t queue, struct rte_ring *out_ring, uint8_t coreI
 		if (unlikely(rx_pkts_number == 0))
 			continue;
 
+		for (i = 0; i < rx_pkts_number; i++) {
+			// TODO refetcht here
+			mbufInit(bufs[i]);
```

Comment:
Make sure that only stores happen inside of the loop, loads or xors are outside of the loop.